WeakAurasOptions/OptionsFrames/OptionsFrame.lua

```diff
@@ -630,9 +630,12 @@ function WeakAuras.CreateFrame()
   unloadedButton:SetViewDescription(L["Toggle the visibility of all non-loaded displays"]);
   frame.unloadedButton = unloadedButton;
 
-  frame.FillOptions = function(self, optionTable)
+  frame.FillOptions = function(self, optionTable, selected)
     AceConfig:RegisterOptionsTable("WeakAuras", optionTable);
     AceConfigDialog:Open("WeakAuras", container);
+    if selected then
+      container.content.obj.children[1]:SelectTab(selected)
```

Comment:
I don't like that. 

I think it's brittle, although it's unlikely that anything will change that breaks it.
But also it's a code path which we won't ever test again, because it's a rather exceptional one.

So imho we should at least mark this with -- TODO: remove

Reply:
I just want to note that this might be handy to investigate for #1208.